Dockerfile

```diff
@@ -4,7 +4,8 @@ LABEL maintainer="https://github.com/tootsuite/mastodon" \
       description="A GNU Social-compatible microblogging server"
 
 ENV RAILS_ENV=production \
-    NODE_ENV=production
+    NODE_ENV=production \
+    SMTP_CA_FILE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
Commit f0fff31 implements the requested changes. I hope I didn't mess up the review workflow by rebasing.